shell/utils/cluster.js

```diff
@@ -63,6 +63,26 @@ export function paginationFilterHiddenLocalCluster(store, filterMgmtCluster = tr
   return PaginationParamFilter.createMultipleFields(filter);
 }
 
+export async function hasAccessToLocalCluster(store) {
```

Comment:
this could just use `management/find`

Reply:
Switched

Reply:
not sure, but i would have hoped fetching an individual resource was highly optimised, even if it does have an extra hop to kube & etcd.

TopLevelMenu has been converted to use SSP (there's some pending stuff from backend though). findPage was intended to only be used in SSP world, mainly for a websocket update reason but soon for others